client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -28,6 +27,19 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		onVideoLoaded: noop,
 	};
 
+	constructor( props ) {
+		super( props );
+
+		this.enableAutoplay = props.isPlaying;
+	}
+
+	// We use a class property because we only want to set this once
```

Comment:
Language nit: "class property" generally refers to static class properties, while "instance properties" are what we're actually doing here. 
```suggestion
	// We use an instance property because we only want to set this once
```